ChangeLog.md

```diff
@@ -24,6 +24,13 @@ See docs/process.md for more on how version tagging works.
   sigaction, sigpending, etc).  We still don't have a way to deliver signals from
   the outside but these at least now work for sending signals to the current
   thread (JS context) (#14883).
+- Remove the workaround used in emcmake and emmake that removed directories
+  with sh.exe from PATH on Windows when MinGW Makefiles generator was used.
+  This was needed with CMake versions older than 3.17.0. If you get an error
+  "sh.exe was found in your PATH" on Windows, please update to CMake 3.17.0
+  or newer (released on March 2020). See
```

Comment:
How about "If you get an error "sh.exe was found in your PATH" on Windows, you can either update to CMake 3.17.0
or newer, or remove the offending directory from your PATH"?